src/index.ts

```diff
@@ -1,33 +1,84 @@
 #!/usr/bin/env node
-import commander from "commander";
+
+import chalk from "chalk";
 import glob from "glob";
+
 import { formatFileStructure } from "./index/formatFileStructure";
+import { version } from "../package.json";
+
+const { argv, env } = process;
+const defaults = {
+  options: {
+    help: false,
+    version: false,
+  },
+  paths: [],
+};
```

Comment:
I'm open for that. I don't see how the `defaults` and etc will be used anywhere else but in the `cli` related code though.

Could you give me a bit more detailed explanation on what your after?

Reply:
Sure, so we will take the `--ignore-pattern` flag as an example.
This flag will need to have some impact of how the [toFractalTree](https://github.com/benawad/butler-cli/blob/master/src/index/formatFileStructure/toFractalTree.ts) function work. Because this function is called by [formatFileStructure](https://github.com/benawad/butler-cli/blob/master/src/index/formatFileStructure.ts#L9) and `formatFileStructure` is called in you `run` function, if we want to give the arg to `toFractalTree` it will need to pass through the args t

Reply:
I'm a bit tired to review code so maybe I'm saying something stupid... Anyway, We will see what Dan has to say about this change!

Reply:
I'm good either way